csharp/ql/lib/semmle/code/csharp/dataflow/LocalCommandInjectionQuery.qll

```diff
@@ -0,0 +1,96 @@
+/**
+ * Provides a taint-tracking configuration for reasoning about local command injection vulnerabilities.
+ */
+
+import csharp
+private import semmle.code.csharp.security.dataflow.flowsources.Local
+private import semmle.code.csharp.frameworks.system.Diagnostics
+private import semmle.code.csharp.security.Sanitizers
+
+/**
+ * A source specific to command injection vulnerabilities.
+ */
+abstract class Source extends DataFlow::Node { }
+
+/**
+ * A sink for command injection vulnerabilities.
+ */
+abstract class Sink extends DataFlow::ExprNode { }
+
+/**
+ * A sanitizer for user input treated as code vulnerabilities.
+ */
+abstract class Sanitizer extends DataFlow::ExprNode { }
+
+/**
+ * DEPRECATED: Use `CommandInjection` instead.
+ *
+ * A taint-tracking configuration for command injection vulnerabilities.
+ */
+deprecated class TaintTrackingConfiguration extends TaintTracking::Configuration {
+  TaintTrackingConfiguration() { this = "LocalCommandInjection" }
+
+  override predicate isSource(DataFlow::Node source) { source instanceof Source }
+
+  override predicate isSink(DataFlow::Node sink) { sink instanceof Sink }
+
+  override predicate isSanitizer(DataFlow::Node node) { node instanceof Sanitizer }
+}
+
```

Comment:
No need to add deprecated configurations

Reply:
Thanks for the feedback, I have removed it. Didn't know if it was required for compatibility or something like that.